social-network/src/your-articles.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    data: PropTypes.objectOf(PropTypes.number), 
+    title: PropTypes.string,
+    img: PropTypes.string,
+}
```

Comment:
These propTypes are never actually being passed/used by the article components. Instead, I would define propTypes for `InnerContent` since you're actually passing/utilizing props in that child component.